composer.json

```diff
@@ -15,6 +15,27 @@
 		"guzzlehttp/promises": "~1.0",
 		"psr/log": "~1.0"
 	},
+	"require-dev": {
+		"phpunit/phpunit": "~4.8.0|~5.3.0",
+		"jakub-onderka/php-parallel-lint": "0.9.2"
+	},
+	"suggest": {
+		"etsy/phan": "Allows running static analysis on the package (requires PHP 7+)"
```

Comment:
Both CI options seem sane to me. Idk a good solution for the non-CI case though. Never seen this situation before

(silently shouts "just switch to PHP 7 already")